web/portal/scripts/test-sync-api-spec

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+API_TYPE=$1
+if [ -z "$API_TYPE" ]; then
+    echo "Usage: $0 [platform|brand|client|user]"
+    exit 1
+fi
+
+# Move to webapp dir
+cd $(dirname "$(realpath -se "$0")")/..
+
+# Generate requested API spec
+/opt/irontec/ivozprovider/web/rest/$API_TYPE/bin/console api:swagger:export > ./cypress/fixtures/apiSpec.json
```

Comment:
This is only a suggestion:
`./bin/console api:swagger:export > ./cypress/fixtures/apiSpec.json
`
Should work fine and we could get rid of the API_TYPE check.

Reply:
The current path is the directory of the portal, and the bin/console must be run in the directory of the rest api application, that's why the full path is specified

p.e: to generate /opt/irontec/ivozprovider/web/portal/platform/cypress/fixtures/apiSpec.json the bin/console script has to be /opt/irontec/ivozprovider/web/rest/platform/bin/console

Reply:
Of course you are right